hardware/Rtl433.cpp

```diff
@@ -343,6 +343,9 @@ bool CRtl433::ParseData(std::map<std::string, std::string>& data)
 
 
 	unsigned int sensoridx = (id & 0xff) | ((channel & 0xff) << 8);
+	if (!model.empty() && (!strcmp(model.c_str(), "TFA-Drop"))) {
```

Comment:
model is a std::string, please do not use the old 'c' method strcmp to search/compare the string.
use == or one of the find functions please